smartsim/_core/mli/comm/channel/dragonfli.py

```diff
@@ -62,11 +62,11 @@ def recv(self) -> t.List[bytes]:
         :returns: the received message"""
         messages = []
         eot = False
-        with self._fli.recvh(timeout=None) as recvh:
+        with self._fli.recvh(timeout=0.001) as recvh:
```

Comment:
OK, I'll do that. Do we think it would be better to expose them as a parameter too (e.g `def recv(self, handle_timeout, receive_timeout)`)?
Asking @AlyssaCote and @ankona too

Reply:
@ankona has an upcoming PR where some timeout parameters are added, I'll punt on this for this PR and open a ticket